source/compatibility.tex

```diff
@@ -1289,7 +1289,7 @@
 \ref{istream::sentry},
 \ref{ostream::sentry},
 \ref{iostate.flags}
-\change Specify use of explicit in existing boolean conversion operators
+\change Specify use of explicit in existing boolean conversion functions
```

Comment:
Yes, but I shouldn't make that change here, which deals with "conversion operator" vs. "conversion function".

I've created #1019 to deal with the "explicit" code font issue.